.github/workflows/main.yml

```diff
@@ -2,82 +2,47 @@ name: CI
 
 on:
   pull_request:
-    branches:
-      - "*"
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
   push:
-    branches:
-      - main
-      - v1
-
+    branches: "main"
 jobs:
-  scan:
-    timeout-minutes: 10
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
-
-  lint:
-    timeout-minutes: 10
+  test:
+    timeout-minutes: 15
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
-
-      - name: Lint code for consistent style
-        run: bundle exec standardrb
-
-  test:
-    timeout-minutes: 10
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
         with:
-          bundler-cache: true
-
-      - name: Install pnpm
-        uses: pnpm/action-setup@v4
-        with:
-          run_install: false
-
-      - name: Setup Node
-        uses: actions/setup-node@v4
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
Logs are showing this value as "cirdes". If I run will this work too?
<img width="253" alt="Screenshot 2025-04-14 at 16 54 35" src="https://github.com/user-attachments/assets/c3a71f74-0fc2-4806-83e2-4d1b8158d31e" />

Reply:
Let’s find this out soon!